### PR TITLE
Update motivation_and_mindset.md

### PR DESCRIPTION
**Pull request by CrypterSr**
## Issue

Closes #XXXXX

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: foundations/introduction/motivation_and_mindset.md [hunk 1]
```diff
@@ -173,6 +173,6 @@ This section contains helpful links to related content. It isn’t required, so
 
 *   [Managing inspiration and motivation](https://markmanson.net/do-something)
 *   [Learning to code when it gets dark](https://medium.freecodecamp.org/learning-to-code-when-it-gets-dark-e485edfb58fd#.yjh0fehje)
-*   [Improve your typing skills with Keybr.com](https://www.keybr.com/). One excellent alternative is [typing.com](https://www.typing.com/). Use these keyboard trainer if you find your typing speed is holding you back. We recommend spending 5 mins every day after you start your PC.
+*   [Improve your typing skills with Keybr.com](https://www.keybr.com/). One excellent alternative is [typing.com](https://www.typing.com/). Another great option is [TypingClub.com](https://www.typingclub.com). Use these keyboard trainer if you find your typing speed is holding you back. We recommend spending 5 mins every day after you start your PC.
```
Comment (zachmmeyer) [inline foundations/introduction/motivation_and_mindset.md:176]: What do you think of this? It gets the main point of the bullet across and keeps all the suggestions in their own sub-bullets.
```suggestion
*   If you find your typing speed is holding you back, you can improve your typing skills by utilizing these keyboard trainers for at least 5 minutes every day:
  * [Keybr.com](https://www.keybr.com/)
  * [typing.com](https://www.typing.com/)
  * [TypingClub.com](https://www.typingclub.com)
```

**Review by zachmmeyer — CHANGES_REQUESTED** [review]
I absolutely love TypingClub. I didn't know it existed when I suggested KeyBr honestly, and I would much prefer we shilled them haha.
I have one recommendation, I'm interested to know what you think.

File: foundations/introduction/motivation_and_mindset.md (hunk 1 above)
Comment (zachmmeyer) [reply foundations/introduction/motivation_and_mindset.md:176]: Good call, I'll update my comment. I didn't notice.

**Review by zachmmeyer — CHANGES_REQUESTED** [review]

**Comment by ManonLef** [comment]
@CrypterSr I noticed you closing and reopening this PR. Let us know if you need any help on how to proceed with the reviews/suggestions.

File: foundations/introduction/motivation_and_mindset.md [hunk 2]
```diff
@@ -173,6 +173,9 @@ This section contains helpful links to related content. It isn’t required, so
 
 *   [Managing inspiration and motivation](https://markmanson.net/do-something)
 *   [Learning to code when it gets dark](https://medium.freecodecamp.org/learning-to-code-when-it-gets-dark-e485edfb58fd#.yjh0fehje)
-*   [Improve your typing skills with Keybr.com](https://www.keybr.com/). One excellent alternative is [typing.com](https://www.typing.com/). Use these keyboard trainer if you find your typing speed is holding you back. We recommend spending 5 mins every day after you start your PC.
-*   [Practice typing with Monkeytype!](https://monkeytype.com/) A minimalistic, customizable typing website. Test yourself in various modes, track your progress and improve your typing speed. It has a great community and frequently receives new features. Even though it has so many features, the experience is still very polished. 
+*   If you find your typing speed is holding you back, you can improve your typing skills by utilizing these keyboard trainers for at least 5 minutes every day:
+  * [Keybr.com](https://www.keybr.com/)
+  * [typing.com](https://www.typing.com/)
+  * [TypingClub.com](https://www.typingclub.com)
+  * [Monkeytype](https://monkeytype.com/)
```
Comment (ManonLef) [inline foundations/introduction/motivation_and_mindset.md:180]:
```suggestion
-   If you find your typing speed is holding you back, you can improve your typing skills by utilizing these keyboard trainers for at least 5 minutes every day:
   - [Keybr.com](https://www.keybr.com/)
    - [typing.com](https://www.typing.com/)
    - [TypingClub.com](https://www.typingclub.com)
    - [Monkeytype](https://monkeytype.com/)
```

**Comment by ManonLef** [comment]
Since there was no response from the issue opener, I have made the suggested changes and also updated the entire list to the correct syntax for you to review @zachmmeyer

**Review by zachmmeyer — APPROVED** [review]
Looks like the merge conflict needs to be resolved and this will be good.
Thanks for finishing this.

**Comment by ManonLef** [comment]
@zachmmeyer thanks, I missed the check :)

**Pull request merged**